sim/visuals/microbit.ts

```diff
@@ -315,6 +315,7 @@ path.sim-board {
         private soundLevel: SVGRectElement;
         private soundLevelText: SVGTextElement;
         private shakeButton: SVGCircleElement;
+        private shakeInitialized: boolean = false;
```

Comment:
On average MakeCode seems to not explicitly type these trivially inferred fields so let's go with:

```suggestion
        private shakeInitialized = false;
```